textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/DictionaryMatcherTest.java

```diff
@@ -64,7 +64,8 @@ public void setUp() throws Exception {
         dataWriter = new DataWriter(dataStore, analyzer);
         QueryParser luceneQueryParser = new QueryParser(TestConstants.ATTRIBUTES_PEOPLE[0].getFieldName(), analyzer);
         luceneQuery = luceneQueryParser.parse(DataConstants.SCAN_QUERY);
-        dataReaderPredicate = new DataReaderPredicate(dataStore, luceneQuery);
+        dataReaderPredicate = new DataReaderPredicate(dataStore, luceneQuery,DataConstants.SCAN_QUERY,
```

Comment:
General comment: please add space at the right place to have a consistent coding style.  For instance: `,DataConstants.SCAN_QUERY` -> `, DataConstants.SCAN_QUERY`.